app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -68,87 +68,62 @@ class GithubWebhook extends RequestHandler<Body> {
   }
 
   Future<void> _handlePullRequest(String rawRequest) async {
-    final PullRequestEvent event = await _getPullRequest(rawRequest);
-    final List<IssueLabel> existingLabels =
-        _getProperty<List<dynamic>>(rawRequest, 'labels')
-            .cast<Map<String, dynamic>>()
-            .map<IssueLabel>(IssueLabel.fromJSON)
-            .toList();
-    final bool isDraft = _getProperty(rawRequest, 'draft');
+    final PullRequest event = await _getPullRequest(rawRequest);
     if (event == null) {
       throw const BadRequestException('Expected pull request event.');
     }
+    final String eventAction = _getEventProperty(rawRequest, 'action');
+
     // See the API reference:
     // https://developer.github.com/v3/activity/events/types/#pullrequestevent
     // which unfortunately is a bit light on explanations.
-    switch (event.action) {
+    switch (eventAction) {
       case 'closed':
         // On a successful merge, check for gold.
         // If it was closed without merging, cancel any outstanding tryjobs.
         // We'll leave unfinished jobs if it was merged since we care about those
         // results.
-        if (event.pullRequest.merged) {
-          await _checkForGoldenTriage(
-            event,
-            existingLabels,
-          );
+        if (event.merged) {
+          await _checkForGoldenTriage(eventAction, event, event.labels);
         } else {
-          await _cancelLuci(
-            event.repository.name,
-            event.number,
-            event.pullRequest.head.sha,
-            'Pull request closed',
-          );
+          await _cancelLuci(event.head.repo.name, event.number, event.head.sha,
+              'Pull request closed');
         }
         break;
       case 'edited':
         // Editing a PR should not trigger new jobs, but may update whether
         // it has tests.
-        await _checkForLabelsAndTests(event, isDraft);
+        await _checkForLabelsAndTests(eventAction, event);
         break;
       case 'opened':
       case 'ready_for_review':
       case 'reopened':
         // These cases should trigger LUCI jobs.
-        await _checkForLabelsAndTests(event, isDraft);
-        await _scheduleIfMergeable(
-          event,
-          labels: existingLabels,
-        );
+        await _checkForLabelsAndTests(eventAction, event);
+        await _scheduleIfMergeable(event);
         break;
       case 'labeled':
         // This should only trigger a LUCI job for flutter/flutter right now,
         // since it is in the needsCQLabelList.
-        if (needsCQLabelList
-            .contains(event.repository.fullName.toLowerCase())) {
-          await _scheduleIfMergeable(
-            event,
-            labels: existingLabels,
-          );
+        if (needsCQLabelList.contains(event.head.repo.fullName.toLowerCase())) {
```

Comment:
Please fix formatting.

Reply:
the if "{" moving at the beginning of the line

Reply:
The left curly bracket has been on the same line with the if.